frame/recovery/src/lib.rs

```diff
@@ -465,21 +464,25 @@ pub mod pallet {
 			ensure!(threshold >= 1, Error::<T>::ZeroThreshold);
 			ensure!(!friends.is_empty(), Error::<T>::NotEnoughFriends);
 			ensure!(threshold as usize <= friends.len(), Error::<T>::NotEnoughFriends);
-			let max_friends = T::MaxFriends::get() as usize;
-			ensure!(friends.len() <= max_friends, Error::<T>::MaxFriends);
-			ensure!(Self::is_sorted_and_unique(&friends), Error::<T>::NotSorted);
+			let bounded_friends: BoundedVec<_, _> =
```

Comment:
```suggestion
			let bounded_friends: FriendsOf<T> =
```

a bit easier to read.